src/rdkafka_cgrp.c

```diff
@@ -95,6 +139,19 @@ rd_kafka_cgrp_max_poll_interval_check_tmr_cb (rd_kafka_timers_t *rkts,
 	 (rkcg)->rkcg_join_state ==				\
 	 RD_KAFKA_CGRP_JOIN_STATE_WAIT_REVOKE_REBALANCE_CB)
 
+/**
+ * @returns true if a rebalance is in progress.
+ */
+#define RD_KAFKA_CGRP_WAIT_REBALANCE(rkcg)                      \
+	((rkcg)->rkcg_join_state ==				\
```

Comment:
yeah, i initially had that but the compiler didn't like what i wrote, so i just wrote it out again :-). I'm leaving this as is in this pass because my plan is to replace this logic with an explicit boolean variable.